Kickstarter-iOS/Views/Controllers/ManagePledgeSummaryViewController.swift

```diff
@@ -12,6 +12,10 @@ final class ManagePledgeSummaryViewController: UIViewController {
   private lazy var backerNumberLabel: UILabel = { UILabel(frame: .zero) }()
   private lazy var backingDateLabel: UILabel = { UILabel(frame: .zero) }()
 
+  private lazy var pledgeStatusLabelViewController: PledgeStatusLabelViewController = {
```

Comment:
Just curious, why did you opt for using a VC here instead of a `UIView`? 🤔

Reply:
I just had a good feeling about it 😄